frontend/src/components/bucket/BucketPermission.vue

```diff
@@ -30,17 +31,32 @@ const updateBucketPermission = (value: any, userId: string, permCode: string) =>
 const removeBucketUser = (userId: string) => {
   bucketStore.removeBucketUser(props.bucketId, userId);
 };
+
+const displayBucketPermissionUser : any = ref(false);
+
+const cancelBucketPermissionUser = () => {
+  displayBucketPermissionUser.value = false;
+};
 </script>
 
 <template>
   <div>
-    <Button class="mt-1 mb-4">
+    <Button
```

Comment:
i wonder if you want this button to be in the `BucketPermissionUser` component template, and have the show/hide logic in that component to keep it completely contained and might avoid some of the emitting. That might not be possible but maybe something to consider.

Reply:
Yeah, that would be better (pls correct me if wrong) but it seems like emit is the new way of two-way binding, props is parent-child only.

Reply:
It's worth considering if this kind of behavior appears in multiple places.